src/QMCHamiltonians/OperatorBase.cpp

```diff
@@ -35,6 +35,10 @@ OperatorBase::OperatorBase() : myIndex(-1), Dependants(0), Value(0.0), tWalker(0
   UpdateMode.set(PRIMARY, 1);
 }
 
+OperatorBase::Return_t OperatorBase::evaluateDeterministic(ParticleSet& P)
+{
+  return evaluate(P);
```

Comment:
Is this a safe fallback? The fallback seems non-deterministic.

Reply:
Unfortunately, the correct behavior of this routine requires estimators with nondeterministic components in their evaluate() have this method overloaded.  Provided that is done, it's safe.  Is this what you were referring to?

Reply:
"the correct behavior of this routine requires estimators with nondeterministic components in their evaluate() have this method overloaded" this piece of info should be recorded in the OperatorBase.h before the function.